### ISSUE TITLE
[MAS2.4.7][Keyboard Navigation -Open bot dialog] Visual focus indicator is not visible on Close button(X) 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1726]
**Actual Result:**
Visual focus indicator is not visible on Close button(X) which is present in the open bot dialog

**Expected Result:**
Visual focus indicator should be visible on Close button(X) 

**Note:** this issue exits for all dialog controls Close button 
**Note 1:** Same issue repro for below path 
Welcome bot Framework emulator ->Navigate to the Sign in with Azure account -> Sign in with Azure account dialog
1. Close
Welcome Bot Framework Emulator ->Create new bot configuration
1.Close button
Welcome->Settings 
1.Close button. Etc

**Repro Steps:**
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the Welcome tab screen
3.	Navigate to the open bot control 
4.	Open bot dialog gets opened and Navigate to the close button using tab key 
5.	Observe the issue

**User Impact:**
users who rely on Keyboard  navigation will be affected the keyboard focus indicator is not visible on  control  so that keyboard dependant users will be affected  to locate  where exact position of keyboard focus.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)
[MAS2.4.7_Open bot dialog.zip](https://github.com/microsoft/BotFramework-Emulator/files/3500928/MAS2.4.7_Open.bot.dialog.zip)
![MAS2 4 7_Open bot dialog](https://user-images.githubusercontent.com/53518994/63014341-f031ee80-beab-11e9-97bb-9e3bdbfa686c.png)
